File: ru.kmorozov.api.onedrive/src/main/java/ru/kmorozov/onedrive/client/facets/ErrorFacet.java
package ru.kmorozov.onedrive.client.facets;

import com.google.api.client.util.Key;

public class ErrorFacet {

    @Key
    private String code;
    @Key
    private String message;
    @Key
    private ErrorFacet innerError;

    public String getCode() {
        return this.code;
    }

    public String getMessage() {
        return this.message;
    }

    public ErrorFacet getInnerError() {
        return this.innerError;
    }
}

File: ru.kmorozov.api.onedrive/src/main/java/ru/kmorozov/onedrive/client/facets/ErrorFacet.kt
package ru.kmorozov.onedrive.client.facets

import com.google.api.client.util.Key

class ErrorFacet {

    @Key
    val code: String? = null
    @Key
    val message: String? = null
    @Key
    val innerError: ErrorFacet? = null
}
